File: RestaurantsApi/Models/Order.cs

using System;
using System.Collections.Generic;
namespace RestaurantsApi.Models
{
    public class Order
    {
        public string BillId { get; set; }
        public string OrderId { get; set; }
        public string TableNumber { get; set; }
        public List<Food> FoodOrder { get; set; }
        public int AmountCustomer { get; set; }
        public int MoneyReceived { get; set; }
        public int TotalMoneyOrder { get; set; }
        public int MoneyCommute { get; set; }
        public int DiscountPersen { get; set; }
        public int DiscountBath { get; set; }
        public int MoneyDiscount { get; set; }
        public int MoneyDiscountTotal { get; set; }
        public string OrderDate { get; set; }
        public string BillTime { get; set; }
        public string OrderStatus { get; set; }
        public string OrderStatusTotal { get; set; }
        public string OrderStatusPayment { get; set; }
        public string OrderStatusFood { get; set; }
        public string OrderStatusDrink { get; set; }
        public string OrderFoodType {get;set;}
        public List<User> OrderReceived { get; set; }

    }
}

File: RestaurantsApp/src/app/Models/order.ts
import { food } from "../Models/food";
import { user } from "../Models/user";

export class order {
    billId: string;
    orderId: string;
    tableNumber: string;
    foodOrder: food[];
    amountCustomer: number;
    moneyReceived: number;
    totalMoneyOrder: number;
    moneyCommute: number;
    discountPersen:number;
    discountBath:number;
    moneyDiscount:number;
    orderDate: string;
    billTime:string;
    orderStatus: string;
    orderStatusTotal:string;
    orderStatusPayment: string;
    orderStatusFood: string;
    orderStatusDrink:string;
    orderReceived: user[];

}